ark/utils/plot_utils.py

```diff
@@ -6,10 +6,48 @@
 import skimage.io as io
 import matplotlib as mpl
 import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
 
 from skimage.segmentation import find_boundaries
 from skimage.exposure import rescale_intensity
 
+from ark.utils import misc_utils
+
+# plotting functions
+
+
+def plot_clustering_result(img_list, titles, save_dir=None, cmap='tab20',
```

Comment:
Given that this is going to be run in a jupyter notebook, I don't think sequentially plotting all (potentially 100+) labels makes sense. Instead, let's allow the user to pass in the name of a specific FOV with the relabel_xr (see my comment from above) and plot that one

Reply:
changed